shared/css/color.scss

```diff
@@ -6,6 +6,7 @@
 
 $black: #000;
 $background_black: #121416;
+$darkest_slate_gray: #25292d;
```

Comment:
NIT: It looks like the convention in this file is to order items by light -> lighter -> lightest and dark -> darker -> darkest. Because of that, this would fit better underneath `$dark_slate_gray`

Reply:
Sure, was matching this file so @sanchitmalhotra126 FYI -- should probably make same change to avoid merge conflicts:

https://github.com/code-dot-org/code-dot-org/pull/44643/files